sql_generators/glean_usage/templates/event_monitoring_aggregates_v1.query.sql

```diff
@@ -69,7 +69,26 @@
       normalized_country_code,
       client_info.app_channel AS channel,
       client_info.app_display_version AS version,
-      ping_info
+      STRUCT (
+        ping_info.end_time,
+        ARRAY(
+          SELECT AS STRUCT
+            key,
+            STRUCT(
+              value.branch,
+              STRUCT(
+                value.extra.type,
+                value.extra.enrollment_id
+              ) AS extra
+            ) AS value
+          FROM
+            UNNEST(ping_info.experiments)
```

Comment:
Note that unnesting isn't guaranteed to preserve the order of the unnested elements.  As long as the order of the experiements array elements doesn't matter this is fine/simpler/preferable, but if order matters you'll need to use `WITH OFFSET ...` and `ORDER BY` to preserve the order.

Reply:
That's a good point.  I'm pretty sure order doesn't matter but I'll add it